apps/website/src/app/playground/ClientPlayground.tsx

```diff
@@ -102,6 +103,9 @@ export const ClientPlayground: FC = () => {
           </div>
         )}
         onCodeChange={(code) => updateUrl({ code })}
+        linker={getHubLinker(squiggle, {
+          hubServer: process.env["NEXT_PUBLIC_HUB_SERVER"],
+        })}
```

Comment:
If the env variable is not unset, it will default to the production squigglehub.org.

The behavior of the playground is slightly different from the hub's editor: there are no import tooltip cards. Those are still done on the hub's side. It should be possible to bring them over (by extracting to some common internal-packages library) later.

<img width="946" alt="Screenshot 2025-03-15 at 11 46 36" src="https://github.com/user-attachments/assets/35b2de1e-f8cc-45b4-bb00-7d9b71b11b2f" />